src/plugins/simulator/simulations/potentiometer-servo/chip-sim-utils.tsx

```diff
@@ -221,22 +225,22 @@ export function getMiniNodesDisplayData(programData?: SharedProgramDataType): IM
       inputNodesArr: [],
       operatorNodesArr: [],
       outputNodesArr: [],
-      extraCount: 0
+      extraInputCount: 0,
+      extraOperatorCount: 0,
+      extraOutputCount: 0
     };
   }
   const arr = [...programData.programNodes.values()];
 
   const formattedData = arr.map(node => {
     const { truncatedLabel, iconKey } = getMiniNodeLabelContent(node);
     const val = node.nodeValue;
-    const valAsNum = Number.isInteger(val) ? val : val.toFixed(2);
-    const valAsString = valAsNum.toString();
 
     return {
       id: node.id,
       iconKey,
       label: truncatedLabel,
-      value: valAsString,
+      value: val.split(" ")[0], // e.g. 100% closed is just 100%
```

Comment:
This logic for processing the value should be moved into `getSharedProgramNodeValue()` that I described above. This way nodes can provide values with spaces in the future.